sdk/storage/Azure.Storage.Common/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 12.6.0-preview.2 (Unreleased)
-
+## 12.6.0 (2020-11-10)
+- This release contains bug fixes to improve quality.
```

Comment:
Weren't there new SAS features added to Common?

Reply:
The only new thing added to Common was a new constructor for `AccountSasBuilder`, so `AccountSasBuilder(permissions, expiresON, services, resourceTypes)`. It was more for ease of the customer knowing what's the minimum requirements to create a SAS and to construct the SASs from the Service Clients. It's not really a new feature.